resources/function_help/json/if

```diff
@@ -4,10 +4,11 @@
   "description": "Tests a condition and returns a different result depending on the conditional check.",
   "arguments": [
 	{"arg":"condition","description":"the condition which should be checked"},
-	{"arg":"result_when_true","description":"the result which will be returned when the condition is True"},
-	{"arg":"result_when_false","description":"the result which will be returned when the condition is False"}
+	{"arg":"result_when_true","description":"the result which will be returned when the condition returns True or not null value"},
```

Comment:
A condition does not "return" (a function does), a condition "is" or "evaluates to". To improve the docs here it would IMO be better to be more precise on what True means (like done in the second example). In general it's also not "not null value" but "not null".